components/core/tools/docker-images/clp-core-ubuntu-noble/Dockerfile

```
@@ -0,0 +1,29 @@
+FROM ubuntu:noble AS base
+
+# Install runtime dependencies
+# TODO: Investigate why libssl-dev is a hidden dependency of clp-s
+RUN apt-get update \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)